docs/Networking.asciidoc

```diff
@@ -52,6 +52,102 @@ or Wicked way:
 </interface>
 ---------------
 
+## TAP with Open vSwitch
+
+The recomended way to configure the network for TAP devices is using Open vSwitch.
+There is a support service +os-autoinst-openvswitch.service+ which sets vlan number
+of Open vSwitch ports based on +NICVLAN+ variable - this separates the groups of
+tests from each other.
+Compared to VDE setup discussed later, Open vSwitch is more complicated to configure,
+but provides more robust and scalable network.
+
+Starting Open vSwitch and adding TAP devices:
```

Comment:
Why present continuous tense? 'Start Open vSwitch and add TAP devices:' would be better, I think